src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py

```diff
@@ -75,3 +85,118 @@ def load_arguments(self, _):
 
     with self.argument_context('cosmosdb database') as c:
         c.argument('throughput', type=int, help='Offer Throughput (RU/s)')
+
+# SQL database
+    with self.argument_context('cosmosdb sql database') as c:
+        c.argument('account_name', options_list=['--account-name', '-a'], help="Cosmosdb account name", id_part=None)
+        c.argument('database_name', options_list=['--name', '-n'], help="Database name")
+        c.argument('throughput', help='The throughput of SQL database (RU/s). Default value is 400')
+
+# SQL container
+    with self.argument_context('cosmosdb sql container') as c:
+        c.argument('account_name', options_list=['--account-name', '-a'], help="Cosmosdb account name", id_part=None)
+        c.argument('database_name', options_list=['--database-name', '-d'], help="Database name")
+        c.argument('container_name', options_list=['--name', '-n'], help="Container name")
+        c.argument('partition_key_path', options_list=['--partition-key-path', '-p'], help='Partition Key Path, e.g., \'/address/zipcode\'')
+        c.argument('default_ttl', options_list=['--ttl'], type=int, help='Default TTL. If the value is missing or set to "-1", items don’t expire. If the value is set to "n", items will expire "n" seconds after last modified time.')
+        c.argument('indexing_policy', options_list=['--idx'], type=shell_safe_json_parse, completer=FilesCompleter(), help='Indexing Policy, you can enter it as a string or as a file, e.g., --idx @policy-file.json or ' + SQL_GREMLIN_INDEXING_POLICY_EXAMPLE)
```

Comment:
FilesCompleter seems not working on windows with @ prefix. I can type the path first with completion feature, then add @ in front of the path though. But this is not so convenient. I'm not sure if I'm missing the correct way to use it.